model/file.go

```diff
@@ -5,11 +5,18 @@ import (
 	"strings"
 )
 
+const (
+	FILETYPE_DOWNLOAD   = 0
+	FILETYPE_ATTACHMENT = 1
```

Comment:
This is how we emulated enums in go before:

```suggestion
	FILETYPE_DOWNLOAD   = iota + 1 // <- this resolves to 1
	FILETYPE_ATTACHMENT // <- and this becomes 2
```